system-x/services/graphql/src/main/java/software/tnb/splunk/service/GraphQL.java

```diff
@@ -0,0 +1,27 @@
+package software.tnb.splunk.service;
+
+import software.tnb.common.account.NoAccount;
+import software.tnb.common.client.NoClient;
+import software.tnb.common.service.Service;
+import software.tnb.common.validation.NoValidation;
+
+public abstract class GraphQL extends Service<NoAccount, NoClient, NoValidation> {
```

Comment:
add `implements WithDockerImage`

Reply:
Ok, what is it needed for ?

Reply:
it adds the possibility to override the image using a system property, in this case using `tnb.graphql.image`: https://github.com/tnb-software/TNB/blob/main/system-x/common/src/main/java/software/tnb/common/deployment/WithDockerImage.java#L15-L18